src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -57,4 +62,14 @@ public static ReadOnlyAddressBook getSampleAddressBook() {
                 .collect(Collectors.toSet());
     }
 
+    public static Cheese[] getSampleCheeses() {
+        return new Cheese[]{
+            new Cheese(new CheeseType("Brie"), new ManufactureDate("2021-03-10"),
```

Comment:
Use the static method `getCheeseType` instead of the constructor (wanted to make the CheeseType a bit Enum like so we can also use `==` for comparison)

Reply:
ahh I get what you mean. Made the change